packages/astro/package.json

```diff
@@ -214,8 +214,11 @@
     "srcset-parse": "^1.1.0",
     "unified": "^10.1.2"
   },
+  "peerDependencies": {
+    "sharp": "^0.32.1"
+  },
```

Comment:
Sharp is already a dep of Astro, I think the issue is moreso with how Sharp is loaded in the image service, isn't it?

Reply:
Oh yeah, mah bad didn't see that it was already in the dep.